codersdk/deployment.go

```diff
@@ -3030,6 +3039,43 @@ Write out the current server config as YAML to stdout.`,
 			Hidden:      true, // Hidden because most operators should not need to modify this.
 		},
 		// Push notifications.
```

Comment:
@johnstcn I think this is vestigial; can I remove it?

Reply:
Yes, this can go!